docs/workflow-of-workflows.md

```diff
@@ -0,0 +1,116 @@
+# Workflow of Workflows
+
+> v2.9 and after
+
+## usecase
+Workflow needs to trigger one of more workflows and manage it. 
+ 
+## solution
+`WorkflowTemplateRef` can be used to create workflow with in two line.
+1. Define your workflow as a `workflowtemplate`.
+E.g:
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: workflow-template-whalesay-template
+spec:
+  entrypoint: whalesay-template
+  templates:
+  - name: whalesay-template
+    inputs:
+      parameters:
+      - name: message
+    container:
+      image: docker/whalesay
+      command: [cowsay]
+      args: ["{{inputs.parameters.message}}"]
+---
```

Comment:
Please put each object on its own codeblock (do not use `---` to separate), it's easier to read that way